src/BCrypt/BCrypt.Helpers.cs

```diff
@@ -13,6 +13,11 @@ namespace PInvoke
     public static partial class BCrypt
     {
         /// <summary>
+        /// An array whose content doesn't matter.
+        /// </summary>
+        private static readonly byte[] OneElementDummyArray = new byte[1];
```

Comment:
It's not about whether it's the same reference. At least that wasn't my reason. My reason is that the `fixed` expression is extremely limited in what it can do, and it must point to an element of the array, and one can't do that when there are no elements. :)